lib/robot.rb

```diff
@@ -14,6 +14,7 @@ def initialize(name)
         (name || ENV["NAME"]), [], nil
 
       redis_connection = Redis.new(:host => (ENV["REDIS_HOST"] || "127.0.0.1"),
+                                   :db   => (ENV["REDIS_DB"]   || 15),
```

Comment:
Since DB 0 is the default in Redis when opening new connections with that parameter unspecified, how about we change this default to 0?